app/code/community/Oyst/OneClick/Model/Magento/Quote.php

```diff
@@ -93,10 +96,58 @@ private function initializeQuote($storeId = null)
         Mage::getSingleton('checkout/session')->replaceQuote($this->quote);
     }
 
+    /**
+     * Create new customer.
+     *
+     * @param string $firstname
+     * @param string $lastname
+     * @param  string $email
+     *
+     * @return false|Mage_Core_Model_Abstract
+     */
+    private function createCustomer($firstname, $lastname, $email)
+    {
+        $customer = Mage::getModel('customer/customer');
+        $store = Mage::app()->getStore();
+
+        $customer->setWebsiteId($this->websiteId)
+            ->setStore($store)
+            ->setFirstname($firstname)
+            ->setLastName($lastname)
+            ->setEmail($email)
+            ->setPassword($this->generatePassword());
```

Comment:
use the Magento ```generatePassword()```
https://github.com/OpenMage/magento-mirror/blob/f9188bdc4df444a2c2b34a401375c5d12fbbfd2e/app/code/core/Mage/Customer/Model/Customer.php#L420